src/Executables/Examples/ParallelTutorial/MinimalExecutable.hpp

```diff
@@ -5,8 +5,11 @@
 
 #include <vector>
 
+#include "DataStructures/DataBox/DataBox.hpp"
```

Comment:
this isn't needed (at least in this file...)

Reply:
It's needed somewhere... but yes I agree not here.

Reply:
@kidder I squashed in a fix... instead I now have a forward-decl in `AlgorithmMetafunctions.hpp`